crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -231,7 +333,29 @@ pub fn check_visibility(
                     }
                 }
 
-                computed_visibility.is_visible = true;
+                computed_visibility.is_visible_in_view = true;
+                let cell = thread_queues.get_or_default();
+                let mut queue = cell.take();
+                queue.push(entity);
+                cell.set(queue);
+            },
+        );
+
+        visible_no_aabb_query.par_for_each_mut(
+            1024,
```

Comment:
This should be a named constant, with some justification in the documentation.

Reply:
Making this a constant would make it a lot easier to experiment to adjust for other devices, where maybe this isn't ideal. It would also mark something that might be worth investigating.

Reply:
Added the batch size constant.